src/index.ts

```diff
@@ -333,27 +361,36 @@ async function wranglerCommands() {
 			setOutput("command-output", stdOut);
 			setOutput("command-stderr", stdErr);
 
-			// Check if this command is a workers or pages deployment
+			// Check if this command is a workers deployment
+			if (command.startsWith("deploy") || command.startsWith("publish")) {
+				const { deploymentUrl, aliasUrl } =
+					extractDeploymentUrlsFromStdout(stdOut);
+				setOutput("deployment-url", deploymentUrl);
+				setOutput("deployment-alias-url", aliasUrl);
+			}
+			// Check if this command is a pages deployment
 			if (
-				command.startsWith("deploy") ||
-				command.startsWith("publish") ||
 				command.startsWith("pages publish") ||
 				command.startsWith("pages deploy")
 			) {
-				// If this is a workers or pages deployment, try to extract the deployment URL
-				let deploymentUrl = "";
-				const deploymentUrlMatch = stdOut.match(/https?:\/\/[a-zA-Z0-9-./]+/);
-				if (deploymentUrlMatch && deploymentUrlMatch[0]) {
-					deploymentUrl = deploymentUrlMatch[0].trim();
-					setOutput("deployment-url", deploymentUrl);
-				}
-
-				// And also try to extract the alias URL (since wrangler@3.78.0)
-				const aliasUrlMatch = stdOut.match(
-					/alias URL: (https?:\/\/[a-zA-Z0-9-./]+)/,
+				const pagesArtifactFields = await getDetailedPagesDeployOutput(
+					config.WRANGLER_OUTPUT_DIR,
 				);
-				if (aliasUrlMatch && aliasUrlMatch.length == 2 && aliasUrlMatch[1]) {
-					const aliasUrl = aliasUrlMatch[1].trim();
+
+				if (pagesArtifactFields) {
+					setOutput("id", pagesArtifactFields.deployment_id);
+					setOutput("deployment-url", pagesArtifactFields.url);
+					setOutput("deployment-alias-url", pagesArtifactFields.alias);
+					setOutput("environment", pagesArtifactFields.environment);
```

Comment:
We need to support `deployment-url` and `deployment-alias-url` as this was what is [currently used](https://github.com/cloudflare/wrangler-action?tab=readme-ov-file#using-the-deployment-url-and-deployment-alias-url-output-variables) in wrangler-action.

Need to figure out if `id` and `environment` are what we want to go with ( this is what [pages-action](https://github.com/cloudflare/wrangler-action?tab=readme-ov-file#using-the-deployment-url-and-deployment-alias-url-output-variables )) uses. However, we've been prefixing outputs with `deployment` in wrangler-action